NOTHING landing

VitaliiVashkevych opened the pull request:
[DEMO LINK](https://VitaliiVashkevych.github.io/layout_landing-page/)

stassokolov1 reviewed (changes requested):
Almost done, check please loom link (https://www.loom.com/share/5d344fb25bee4fe7a257992e9b98b85b)🔥

VitaliiVashkevych commented:
1. Додав Hover на іконку телефону (Header)
2. Burger menu - не відкриває нову сторінку
3. Виправив усі шрифти
4. Виправив відступ у кнопки View All
5. Розтягнув зображення над About Us
6. Виправив форми (додав Hover та Focus, виправив placeholder в textarea)
7. Додав поінтер і Hover на кнопки
8. Виправив e-mail (contacts)
9. Додав адресу на Googe Maps (contacts)

roman-mirzoian reviewed (changes requested):
Good job 👍, but not all tasks in the checklist have been completed.
- Use the logo as a favicon.
- All form fields should be required and have correct types (email, tel, etc.)

When clicking on the logo in the header, the link does not work correctly and opens a new window in the browser.
https://gyazo.com/dc63ea82634b4d5c9270310e3dc381c9

VitaliiVashkevych commented:
1. Added favicon
2. All form fields are required and have correct types
3. Removed unnecessaries `target='_blank'` attributes from refs

polosanya reviewed (changes requested):
Almost perfect! Check your page on different devices using DevTools to make sure that it looks great on all of them
1. Pay attention to the design of the mobile version
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/a55ce364-e44e-414b-b810-bc36e24f8308)
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/e6e588ca-0a08-45cb-a37b-f70ec489ad2c)
2. Same problem on tablets
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/03dab1bc-9432-43ae-ac82-56300fc706f4)
3. Also, make sure that all elements are centred
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/58114507-09a7-4049-9078-bdab5c0d31b0)
4. "Send" button should be dark by default and light on hover, just like "View all" button
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/a801f9c4-268f-40a0-a50a-7f357533cb7a)
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/bfedebc8-642c-49e5-84da-fc73efa56806)
5. Would be great to add some hover effects to these links. Also, make sure that indentation between them is correct
![image](https://github.com/mate-academy/layout_landing-page/assets/96409563/37b808f4-b9b2-458b-8a7c-36f255be1a0a)

VitaliiVashkevych commented:
1. Corrected size and indentation of 'View all' button on all devices.
2. Centered all elements
3. All buttons are dark by default now, and light on hover
4. Corrected indentations in bottom links, + added hover effects (font size + underline)
5. Other minor improvements

roman-mirzoian reviewed (approved):
Well done 👍

StasSokolov1 commented:
<img width="501" alt="Screenshot 2024-05-29 at 15 59 20" src="https://github.com/mate-academy/layout_landing-page/assets/75898310/93e1c917-b7a0-4f6d-8fd5-32f1fdb9957b">
